ecs-cli/modules/cli/configure/configure.go

```diff
@@ -29,6 +33,67 @@ func fieldEmpty(field string, flagName string) error {
 	return nil
 }
 
+// Migrate is the callback for the Migrate command
+func Migrate(context *cli.Context) error {
+	oldConfig := &config.CLIConfig{}
+	dest, err := config.NewDefaultDestination()
+	if err != nil {
+		return errors.Wrap(err, "Error reading old config file.")
+	}
+
+	// The iniReadWriter is rather terrible and will not throw parse errors
+	// See if we can read the config file as YAML, if that's true then we don't
+	// need to migrate
+	clusterConfig, err := config.ReadClusterFile(config.ConfigFilePath(dest))
+	if err == nil && clusterConfig.Version != "" {
+		// Is a new style config file
+		logrus.Fatalf("No need to migrate; found a YAML formatted config file at %s", config.ConfigFilePath(dest))
```

Comment:
Do we need to set the exit code to 1 because this is more like no op, not really an error? Should we use logrus.Errorf here instead?